server/config.js

```diff
@@ -1,12 +1,19 @@
+import fs from 'fs';
 import path from 'path';
 
 const root = (dir) => path.resolve(__dirname, '..', dir);
 
+let localConfig = {};
+
+if (fs.existsSync(root('server/config.local.js'))) {
+  localConfig = require('./config.local');
```

Comment:
Since firstly you check if `config.locals.js` exists using absolute path, then you require that file using relative path.  Personally i think you can either using absolute or relative path to keep  consistence.

```javascript
let localConfigFile = root('server/config.local.js');
if (fs.existsSync(localConfigFile)) {
  localConfig = require(localConfigFile);
}
```

Reply:
eslint rule  ``import/no-dynamic-require`` forbidden dynamic require path

Reply:
I will set ``import/no-dynamic-require`` 0 in eslintrc

Reply:
Disable this rule is ok.